Makefile

```diff
@@ -22,6 +22,8 @@ firefox_nm_dir ?= /usr/lib/mozilla/native-messaging-hosts
 chrome_nm_dir ?= /etc/opt/chrome/native-messaging-hosts
 chrome_ext_dir ?= /usr/share/google-chrome/extensions
 chromium_nm_dir ?= /etc/chromium/native-messaging-hosts
+# python3 system interpreter for global installs
+python3_bin ?= /usr/bin/python3
```

Comment:
Why hard-coding the default, why not resolving it during installation (`which python3`)?

Reply:
Because exactly that is problematic. `which python3` considers the current `PATH` (similar to `/usr/bin/env python3`) and that is not what we want here. When users modify their `PATH` variable used in the user session (e.g. via systemd or via `.bashrc`), this must not leak into applications which are installed system wide and call other processes internally.

By running `which python3` at install time, this is even worse as it hard-codes the wrong value for the syswide installation. All major distros I checked have a statement in their policies to hard-code the path to the (distro) Python interpreter in the shebang.

Reply:
Distros know the path they use, but we don't know the distro. If the user does an installation for the system from an unclean context, well... But can't we create a clean one to resolve the path?

Reply:
The path can be controlled by a variable and this is already documented. But we could indeed use `which python3` as a default instead of the `/usr/bin/python3`. For our Debian package target we then set it according to the Debian policy.

Reply:
@jan-kiszka I changed the code to default to `which python3`
@johann-pfefferl That's indeed better. I changed it as well.